docs/_basic/authenticating_oauth.md

```diff
@@ -6,16 +6,54 @@ order: 15
 ---
 
 <div class="section-content">
-To make your Slack app ready for distribution, you will need to implement OAuth and store installation information (i.e. access tokens) securely. Bolt supports OAuth and will handle most of the work for you by setting up OAuth routes and verifying state. 
+To prepare your Slack app for distribution, you will need to enable Bolt OAuth and store installation information securely. Bolt supports OAuth and will handle the rest of the work; this includes setting up OAuth routes, state verification, and passing your app an installation object which you must store. 
 
-You will need to provide your:
-* `clientId`, `clientSecret`, `stateSecret` and `scopes` (required)
-* An `installationStore` option with `storeInstallation` and `fetchInstallation` handlers defined for storing installation data to a database *(optional, but highly recommended for production)*
+To enable OAuth, will need to provide:
+* `clientId`, `clientSecret`, `stateSecret` and `scopes` _(required)_
+* An `installationStore` option with handlers that store and fetch installations to your database *(optional, strongly recommended in production)*
+
+---
+
+##### Development and Testing
```

Comment:
Added a new section here that calls out `FileInstallationStore` as an option for development.